src/auth/subscribers/auth-subscriber.ts

```diff
@@ -1,22 +1,22 @@
 import { InsertEvent, UpdateEvent, EntitySubscriberInterface, EventSubscriber } from 'typeorm';
-import { Authentication } from '../entities';
-import { HashProvider } from '../../utils';
+import { User } from '../../users/entities/user.entity';
+import { AuthService } from '../services/auth.service';
 
 @EventSubscriber()
-export class AuthSubscriber implements EntitySubscriberInterface<Authentication> {
+export class AuthSubscriber implements EntitySubscriberInterface<User> {
```

Comment:
What is the benefit of having EventSubscriber on this class?

Reply:
I've found that my alternatives are either to hash passwords in services or in dtos via decorators. I'll think about what's the best solution

Reply:
I decided to move hashing into services, since it's the simplest solution and I already have getHash function in the service so less imports and files :)